changelog/4321.breaking.1.rst

```diff
@@ -0,0 +1 @@
+Removed `:func:~sunpy.net.dataretriever.QueryResponse.create`.
```

Comment:
Can you check if ":func:" should be outside or inside of the backticks?

Reply:
According to docs
```
python function :py:func:`package.name.function`
:py:meth:`package.name.ClassName.method`.
```
So ideally , we should use `:meth:` here. But we have used both meth and func across sunpy, in changelog and the rst docs. However, in this PR I'll use meth. And func should be outside those ticks, which also need to be changed.

Reply:
Can you use the correct one?

Reply:
Yes, I have used the correct one, in all changelogs now.